src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig

```diff
@@ -175,11 +175,7 @@
         <script>Sfjs.initializeLogsTable();</script>
     {% endif %}
 
-    {% set compilerLogTotal = 0 %}
-    {% for logs in collector.compilerLogs %}
-        {% set compilerLogTotal = compilerLogTotal + logs|length %}
-    {% endfor %}
-
+    {% set compilerLogTotal = collector.compilerLogs|reduce((total, logs) => total + logs|length) %}
```

Comment:
this should use `0` as second argument. Otherwise, the initial value will be `null`, not `0`

Reply:
Not sure if I understand you well. If I replace `logs` by `0` I see this error:

![twig-reduce](https://user-images.githubusercontent.com/73419/209166043-a9932a6c-9790-4ba5-9178-febcadadbeec.png)

Reply:
you pass a callback as first argument of `reduce`. You omit the second one currently.

Reply:
Now I understand you. I added the second argument (as `0`) in all our `|reduce` usages. Thanks!